.gitpod.yml

```diff
@@ -40,6 +40,9 @@ tasks:
   - name: Go
     init: leeway exec --filter-type go -v -- go mod verify
     openMode: split-right
+  - name: pre-commit
+    # install pre-commit hooks into git
+    before: pre-commit install
```

Comment:
It runs on CI in 
https://github.com/gitpod-io/gitpod/blob/39cf522451dfd0516116df6b089620ac60cf6489/.werft/jobs/build/validate-changes.ts#L30-L40

Reply:
Outside of that, I haven't found any other usages of `pre-commit install`. pre-commit itself is installed in the dev Dockerfile.

Reply:
I thought about in IDE, but cannot find it.

Reply:
Ah, doesn't the Git initializer already install all hooks? (I think I saw it fail once because of Git hook errors like a year ago)

Reply:
In my test workspace runs, it didn't install the hooks. I also didn't find any referneces to `pre-commit install` in the codebase so this would need to happen somewhere outside (I don't yet have context where that would be).

Reply:
Also worth noting: I believe the latest Terminal (i.e. `-`) in `tasks` ends up being the one visible by default. So here, you're hiding the right Go Terminal, and showing a Terminal on top that does just `pre-commit install`.

Suggestions:
- Maybe it's better to add `pre-commit install` as a `before` to any other task listed above
- I personally really dislike the split Terminals (TypeScript on left, Go on right) and would also be very happy if we remove the split (but this is unrelated to this PR, and others might really like the split)

Reply:
Thanks for raising this. I will make the requested change here to not change the experience.